db/migrate/20180827225542_create_webauthn_configurations_table.rb

```diff
@@ -0,0 +1,12 @@
+class CreateWebauthnConfigurationsTable < ActiveRecord::Migration[5.1]
+  def change
+    create_table :webauthn_configurations do |t|
+      t.references :user, null: false
+      t.text :public_key, null: false
+      t.boolean :mfa_enabled, default: true, null: false
```

Comment:
I'm not sure if we really need this, but probably not a bad idea. The main reason for adding this to the phone configuration is so we can remember confirmed phones without using them for MFA.